chart/node-policy-webhook/values.yaml

```diff
@@ -21,6 +21,11 @@ certs:
   # Key key.pem
   key: ""
 
+
+# Additional Args to command
+additionalArgs:
```

Comment:
I would rather have `verbosity` than additional args... although this option might be useful in the future

Reply:
you are right

What do you think about removing cobra?

Reply:
If it only complicates things, we can surely remove it